app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -129,4 +151,21 @@ def parse_stack(vapp)
     }
     return uid, new_result
   end
+
+  def parse_image(image, is_public)
+    uid         = image.id
+    name        = image.name
+
+    new_result = {
+      :type                 => ManageIQ::Providers::Vmware::CloudManager::Template.name,
+      :uid_ems              => uid,
+      :ems_ref              => uid,
+      :name                 => name,
+      :vendor               => "vmware",
+      :raw_power_state      => "never",
+      :publicly_available   => is_public
+    }
```

Comment:
@cankarm Do you see any other information we could parse from the catalog item? You can check Amazon (https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb#L175) or Google (https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb#L174) to see examples of data from other providers.